usage/jsgui/src/main/webapp/assets/css/base.css

```diff
@@ -1114,6 +1114,11 @@ tr.app-add-wizard-config-entry {
     margin-bottom: 6px;
     background-color: #F0F0F0;
     border-radius: 5px;
+    overflow-y: scroll;
```

Comment:
This causes the vertical scrollbar on Windows. It's on the element that contains all accordion elements. Looks like it's not really needed? It's not visible on OS X due to the transparent scrollbars.

Not important of course, can be merged as is.